engine/src/gfx/nav/navscreen.cpp

```diff
@@ -1769,7 +1769,7 @@ void NavigationSystem::TranslateCoordinates( QVector &pos,
     // as well as to display system names and make systems
     // highlight their names on mouse hover
     // and also make them clickable for setting jump routes
-    float itemscale = CalculatePerspectiveAdjustment(
+    CalculatePerspectiveAdjustment(
```

Comment:
I **think** this is correct. Call function/method CalculatePerspectiveAdjustment, then throw away the result.

Reply:
Here I am just going by a conversation we had on gitter. Hopefully I will understand more about what I am doing in the future. Thanks for double-checking!